CHANGELOG.next.asciidoc

```diff
@@ -30,37 +29,24 @@ https://github.com/elastic/beats/compare/v7.0.0-alpha2...master[Check the HEAD d
 ==== Bugfixes
 
 *Affecting all Beats*
-- Fix panics when a processor is closed twice {pull}34647[34647]
 
 *Auditbeat*
 
 
 *Filebeat*
 
-- [Azure Logs] Fix authentication_processing_details parsing in sign-in logs. {issue}34330[34330] {pull}34478[34478]
-- Fix dropped events when monitor a beat under the agent and send its `Host info` log entry. {pull}34599[34599]
-- Fix panics when a processor is closed twice {pull}34647[34647]
-- Gracefully handle Windows event channel not found errors in winlog input. {issue}30201[30201] {pull}34605[34605]
-- Fix handling of RFC5988 links' relation parameters by `getRFC5988Link` in HTTPJSON. {issue}34603[34603] {pull}34622[34622]
 
 *Heartbeat*
 
-- Enable nodejs engine strict validation when bundling synthetics. {pull}34471[34471]
 
 *Metricbeat*
 
 
 *Packetbeat*
 
-- Handle truncated DNS records more gracefully. {issue}21495[21495] {pull}28297[28297]
```

Comment:
I think this entry didn't get carried over to the `7.17.10` CHANGELOG?

Reply:
Thanks, added it back.